modules/ztp-configuring-disk-partitioning.adoc

```diff
@@ -8,39 +8,179 @@
 
 Configure disk partitioning for a managed cluster using a `SiteConfig` CR and {ztp-first}. The  disk partition details in the `SiteConfig` CR must match the underlying disk.
 
-[NOTE]
+[IMPORTANT]
 ====
-Use persistent naming for devices to avoid device names such as `/dev/sda` and `/dev/sdb` being switched at every reboot. You can use `rootDeviceHints` to choose the bootable device and then use same device for further partitioning.
+You must complete this procedure at installation time.
 ====
 
 .Prerequisites
 
-* You have installed the OpenShift CLI (`oc`).
+* Install Butane.
 
-* You have logged in to the hub cluster as a user with `cluster-admin` privileges.
+.Procedure
 
-* You have created a Git repository where you manage your custom site configuration data for use with {ztp-first}.
+. Create the `storage.bu` file.
++
+[source,yaml]
+----
+variant: fcos
+version: 1.3.0
+storage:
+  disks:
+  - device: /dev/disk/by-path/pci-0000:01:00.0-scsi-0:2:0:0 <1>
+    wipe_table: false
+    partitions:
+    - label: var-lib-containers
+      start_mib: <start_of_partition> <2>
+      size_mib: <partition_size> <3>
+  filesystems:
+    - path: /var/lib/containers
+      device: /dev/disk/by-partlabel/var-lib-containers
+      format: xfs
+      wipe_filesystem: true
+      with_mount_unit: true
+      mount_options:
+        - defaults
+        - prjquota
+----
+<1> Specify the root disk.
+<2> Specify the start of the partition in MiB. If the value is too small, the installation will fail.
```

Comment:
Try to avoid future tense. Revise sentence to "...the installation fails." Same thing for the next sentence.